maintainers/maintainer-list.nix

```diff
@@ -2725,6 +2725,12 @@
     githubId = 1753498;
     name = "Dejan Lukan";
   };
+  electrified = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add electrified``.